configuration/config-txt/overclocking.md

```diff
@@ -73,22 +73,23 @@ This table lists the default `over_voltage` settings for the various Pi models.
 | --- | --- | --- |
 | Pi 1 | 0 | 1.2V |
 | Pi 2 | 0 | 1.2-1.3125V |
-| Pi 3 | 0 | 1.2-1.3125V |
+| Pi 3 | 0 | 1.2-1.3125V 
+| Pi 4 | 0 | 0.88V |
 | Pi Zero | 6 | 1.35V |
 
 #### Specific to Pi 4B
```

Comment:
Heading needs changed. Probably this should say `Pi 4B, Pi 400 and Compute Module 4` given we've specified specific products further up the page.

Reply:
Didn't we agree on another PR recently to just use "Pi 4" as shorthand for "Pi 4B, Pi 400 and Compute Module 4" ?

Reply:
Yes, but the default ARM clock is different on Pi 400, so Tim has given the Pi 400 its own column in the table. Perhaps the table can be simplified by just having the Pi 4 column headed simply `Pi 4` and for the ARM clock list it as `1500 (1800 on Pi 400)`? Then this heading would just be `Pi 4`.

Reply:
> having the Pi 4 column headed simply Pi 4 and for the ARM clock list it as 1500 (1800 on Pi 400)

Yes, that might make things simpler as it looks like all the other values are the same.

Reply:
Added CM4, Pi400. There's no difference in voltage at 1.5 GHz